wg-advisors/README.md

```diff
@@ -102,4 +109,38 @@ the ASF. Trust is transitive - that is, you may not know all members
 personally, but each member was nominated and elected by people that
 you, in turn, nominated and elected.
 
+### How to initiate PMC <-> Advisor relationship?
+
+There is no formal process for this. As an advisor, you may simply
+start participating in the project's mailing lists (dev and private)
+after introducing yourself and asking if this is fine for you to become
+an Advisor. The PMC must be allowed to say "no thank you" without providing any
+reason, and you must respect that decision, and not offer again unless
+invited.
 
+You can also be invited by the PMC to become an Advisor. When you become
+an Advisor, you should sign yourself up to the list of advisors in this
+repository (TODO: how) and describe your experience, skills and what
+you can bring as an Advisor (TODO: how).
+
+The ASF Board can also - during regular Board project review process - to
+identify a project that could benefit from an Advisor, and ask the PMC
+to consider inviting an Advisor, and cc: dev@community.apache.org mailing 
+list to make the advisors aware of such a project (which might also initiate
+a voluntary Advisor to step up and offer their help). In general the PMC
+should be allowed to decide whether they want an Advisor, and who that
+Advisor could bem but if any Advisor is interested in a project, they
+are encouraged to reach out to the PMC and offer their help.
+
+In neither case, the Board is appointing an Advisor. The Board is only
+asking the PMC to consider inviting an Advisor, and cc: the
+dev@community.apache.org mailing list to give the opportunity for
+the interested Advisors to step up and volunteer to help the project.
+
+### How to stop being an Advisor?
+
+You may stop being an Advisor at any time. You should inform the PMC
+that you are stepping down and remove yourself from being advisor of
+the PMC (TODO: how) You do not need to provide any reason for stepping down,
```

Comment:
> how

A NOTICE or request on the mailing list (dev@ or private@community should be OK IMO). Something like https://lists.apache.org/thread/1k116oyo1fc4kk3vdqg6hfz0x1bpxn21.

Reply:
Clarified that a bit - maybe without the example :). but should be pretty clear now how to do it.